department-of-veterans-affairs/caseflow
# 1515 fix tag deletion handling

Connect #1515 
Connect #1717

## To Test
1. Remove a tag, and see that it is instantly removed from the UI
1. Make the server 500 on the tag deletion endpoint
    1. Remove a tag, see that it is instantly removed from the UI, and then re-appears when the error message shows up

## Out of scope
Scenario where user creates a tag, then deletes that tag before that tag it's been saved.
